src/rust/bitbox02-rust/src/hww/api/ethereum/sign.rs

```diff
@@ -70,16 +133,59 @@ fn parse_erc20(request: &pb::EthSignRequest) -> Option<([u8; 20], BigUint)> {
 }
 
 // fee: gas limit * gas price:
```

Comment:
please extend the docstring, the more detail the better, as this fee stuff in both legacy and eip1559 is not easy to grasp. would be good to mention both legacy and eip1559 and what is displayed for the fee.

example

>             // We show the user the max possible fee, but the actual fee might be lower

but in legacy this is also true (actual gas used could be lower than gas limit), so understanding the difference would be helpful.